configs/_base_/datasets/cityscapes_detection.py

```diff
@@ -54,3 +54,6 @@
         img_prefix=data_root + 'leftImg8bit/test/',
         pipeline=test_pipeline))
 evaluation = dict(interval=1, metric='bbox')
+
+# NOTE: This is for automatically scaling LR, USER CAN'T CHANGE THIS VALUE
+mmdet_official_special_batch_size = 8  # (8 GPUs) x (1 samples per GPU)
```

Comment:
We can set the batch size in specific models.

Reply:
the configs in datasets only contain the dataset arguments.

Reply:
I didn't add this `mmdet_official_special_batch_size ` in all the config, just in the config with `data.samples_per_gpu`, is that OK ?

Reply:
Oh, do you mean i need to skip this directory: `configs/_base_/datasets` ?

Reply:
You can set `default_batch_size` in model configs like [faster_rcnn_r50_fpn_1x_cityscapes.py](https://github.com/open-mmlab/mmdetection/blob/11f3ca2ba61b01e84e425bca2b8c6109e525ae67/configs/cityscapes/faster_rcnn_r50_fpn_1x_cityscapes.py).

Reply:
You can also refer to the metafile.yml in each folder to find the model not training with 16 batch size.

Reply:
Done. I remove the `default_batch_size` out of dataset configs

Reply:
When I read the metafile.yml, I have little confuse.

Take yolact as example, In README.md, is said only using single GPU,
![image](https://user-images.githubusercontent.com/25873202/158053107-8d5f521f-e2a3-4d71-a5df-14ffaa64c834.png)

But in the metafile.yml, It show `8x GPUs`
![image](https://user-images.githubusercontent.com/25873202/158053168-e9772e9f-abdc-408d-8b1e-652328be0a27.png)

and some of it shows that batch size is 8 and another is 64, I totally confuse...
![image](https://user-images.githubusercontent.com/25873202/158053215-8574cde1-b23d-4797-8434-43fcf51f7a68.png)